lib/class.LinfoOutput.php

```diff
@@ -243,6 +243,14 @@ public function htmlOut() {
 	if (!empty($settings['show']['hostname']))
 		$core[] = array($lang['hostname'], $info['HostName']);
 	
+	//Web server
```

Comment:
Make methods for these in class.OS.php and make getter calls to them in class.Linfo.php like with all the other data given.